Classes/New Issue/NewIssueTableViewController.swift

```diff
@@ -205,6 +213,10 @@ final class NewIssueTableViewController: UITableViewController, UITextFieldDeleg
         bodyField.inputAccessoryView = actions
     }
 
+    func updateSubmitButtonState() {
+        navigationItem.rightBarButtonItem?.isEnabled = ( titleText == nil || bodyText == nil ) ? false : true
```

Comment:
Like @BasThomas mentioned, you actually can just leave it as 

`navigationItem.rightBarButtonItem?.isEnabled = !(titleText == nil || bodyText == nil)

Reply:
I see what you did there. 😀